docs/TROUBLESHOOTING.md

```diff
@@ -71,55 +70,6 @@ This could be due to init.defaultBranch being set to something other than
    git config --global init.defaultBranch main
    ```
 
-### Terraform State Snapshot lock
-
-**Error message:**
-
-When running the build for the branch `production` in step 3-networks in your **Foundation Pipeline** the build fails with:
-
-```
-state snapshot was created by Terraform v0.x.x, which is newer than current v0.13.7; upgrade to Terraform v0.x.x or greater to work with this state
-```
-
-**Cause:**
-
-The manual deploy step for the shared environment in [3-networks](../3-networks#deploying-with-cloud-build) was executed with a Terraform version newer than version v0.13.7 used in the **Foundation Pipeline**.
```

Comment:
If a user locally runs using 1.x >  1.0, I believe this can still occur. 
https://www.terraform.io/language/v1-compatibility-promises#upgrading-and-downgrading

Reply:
restored with update to v1.0.0 and new instruction on how to update the terraform image